codec-http/src/main/java/io/netty/handler/codec/http/compression/CompressionOptions.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.codec.http.compression;
+
+/**
+ * {@link CompressionOptions} provides compression options for
+ * various types of compressor types, like Brotli.
+ *
+ * A {@link CompressionOptions} instance is thread-safe
+ * and should be shared between multiple instances of Compressor.
+ */
+public interface CompressionOptions {
```

Comment:
I wonder if we should maybe move this to the `codec` module under the `compression` package... @trustin WDYT ?